test/image/mocks/gl2d_22.json

```diff
@@ -1,536 +0,0 @@
-            "ticklen": 5,
-            "tickwidth": 1,
-            "tickcolor": "#444",
-            "tickangle": "auto",
-            "tickfont": {
-                "family": "",
-                "size": 0,
-                "color": ""
-            },
-            "exponentformat": "B",
-            "showexponent": "all",
-            "gridcolor": "#eee",
-            "gridwidth": 1,
-            "zerolinecolor": "#444",
-            "zerolinewidth": 1,
-            "linecolor": "#444",
-            "linewidth": 1,
-            "anchor": "y",
-            "position": 0,
-            "mirror": false,
-            "overlaying": false,
-            "autorange": true
-        },
-        "yaxis": {
-            "title": "Click to enter Y axis title",
-            "titlefont": {
-                "color": "",
-                "family": "",
-                "size": 0
-            },
-            "range": [
-                -0.07433281386749718,
-                1.4817768838674972
-            ],
-            "domain": [
-                0,
-                1
-            ],
-            "type": "linear",
-            "rangemode": "normal",
-            "showgrid": true,
-            "zeroline": true,
-            "showline": false,
-            "autotick": true,
-            "nticks": 0,
-            "ticks": "outside",
-            "showticklabels": true,
-            "tick0": 0,
-            "dtick": 0.2,
-            "ticklen": 5,
-            "tickwidth": 1,
-            "tickcolor": "#444",
-            "tickangle": "auto",
-            "tickfont": {
-                "family": "",
-                "size": 0,
-                "color": ""
-            },
-            "exponentformat": "B",
-            "showexponent": "all",
-            "gridcolor": "#eee",
-            "gridwidth": 1,
-            "zerolinecolor": "#444",
-            "zerolinewidth": 1,
-            "linecolor": "#444",
-            "linewidth": 1,
-            "anchor": "x",
-            "position": 0,
-            "mirror": false,
-            "overlaying": false,
-            "autorange": true
-        },
-        "legend": {
-            "x": 1.02,
-            "y": 1,
-            "traceorder": "normal",
-            "font": {
-                "family": "",
-                "size": 0,
-                "color": ""
-            },
-            "bgcolor": "#fff",
-            "bordercolor": "#444",
-            "borderwidth": 0,
-            "xanchor": "left",
-            "yanchor": "top"
-        },
-        "margin": {
-            "l": 80,
-            "r": 80,
-            "b": 80,
-            "t": 100,
-            "pad": 0,
-            "autoexpand": true
-        },
-        "paper_bgcolor": "#fff",
-        "plot_bgcolor": "#fff",
-        "hovermode": "x",
-        "dragmode": "zoom",
-        "barmode": "group",
-        "bargap": 0.2,
-        "bargroupgap": 0,
-        "boxmode": "overlay",
-        "separators": ".,",
-        "hidesources": false
-    }
-}
```

Comment:
With lots of test mock removals, aren't they going to be missed, or are they covering things covered in other test cases?